aws/retry/jitter_backoff_test.go

```diff
@@ -38,7 +38,7 @@ func TestExponentialJitterBackoff_AttemptDelay(t *testing.T) {
 		"max delay": {
 			MaxBackoff: 20 * time.Second,
 			RandFloat:  func() (float64, error) { return maxB, nil },
-			Attempt:    1 << 53,
+			Attempt:    math.MaxInt,
```

Comment:
**fix**:

`math.MaxInt` was only [added](https://go.dev/doc/go1.17) in Go 1.17. Probably just change this to `int64` type explicitly or change the value to be within range